datadog/resource_datadog_security_monitoring_default_rule.go

```diff
@@ -97,6 +118,19 @@ func resourceDatadogSecurityMonitoringDefaultRuleRead(d *schema.ResourceData, me
 		}
 	}
 
+	ruleFilters := make([]interface{}, len(ruleResponse.GetFilters()))
+	for idx := range ruleResponse.GetFilters() {
+		ruleFilter := make(map[string]interface{})
+		responseRuleFilter := ruleResponse.GetFilters()[idx]
+
+		ruleFilter["action"] = responseRuleFilter.GetAction()
+		ruleFilter["query"] = responseRuleFilter.GetQuery()
+
+		ruleFilters[idx] = ruleFilter
+	}
```

Comment:
Nit: This might make it a bit more concise.

```suggestion
	ruleFilters := make([]map[string]interface{}, len(ruleResponse.GetFilters()))
	for idx, responseRuleFilter := range ruleResponse.GetFilters() {
		ruleFilters[idx] = map[string]interface{}{
			"action": responseRuleFilter.GetAction(),
			"query": responseRuleFilter.GetQuery(),
		}
	}
```